doc/man3/EVP_EncryptInit.pod

```diff
@@ -905,6 +905,16 @@ A getter that returns 1 if the operation is FIPS approved, or 0 otherwise.
 This may be used after calling a cipher final operation such as
 EVP_EncryptFinal_ex(). It may return 0 if the "encrypt-check" option is set to 0.
 
+=item "iv-generated" (B<OSSL_CIPHER_PARAM_AEAD_IV_GENERATED>) <unsigned integer>
+
+An indicator that returns 1 if an IV was generated internally during encryption,
+or O otherwise.
+This may be used by AES GCM ciphers after calling a cipher final operation such
+as EVP_EncryptFinal_ex().
+AES GCM should generate an IV internally if the iv is not specified during a
```

Comment:
Keep case consistent, it's `IV` not `iv`.

Also drop AES here if dropped above.

Reply:
You changed them all to lower case which is wrong.  It should be IV.
There were both in this line, hence the confusion.